src/pages/gd/Bridge/index.tsx

```diff
@@ -15,11 +15,24 @@ import useSendAnalyticsData from 'hooks/useSendAnalyticsData'
 import { KimaModal } from '@gooddollar/good-design'
 
 const Bridge = memo(() => {
-    const { library } = useActiveWeb3React()
+    const { library, chainId } = useActiveWeb3React()
     const [theme] = useApplicationTheme()
     const sendData = useSendAnalyticsData()
     const { switchNetwork } = useSwitchNetwork()
     const [bridgeStatus, setBridgeStatus] = useState<boolean | undefined>(undefined)
+    const activeChain = useRef({ origin: '', destination: '' })
+
+    useEffect(() => {
+        const supportedNetworks = Object.keys(SupportedV2Networks)
+
+        if (chainId === (SupportedV2Networks.CELO as number)) {
+            supportedNetworks.reverse()
+        }
+
+        const networks = { origin: supportedNetworks[0], destination: supportedNetworks[1] }
+
+        activeChain.current = networks
+    }, [/* used */ chainId, activeChain])
```

Comment:
replace with useMemo

```javascript
const activeChain = useMemo(() => {  
         const supportedNetworks = Object.keys(SupportedV2Networks)

         if (chainId === (SupportedV2Networks.CELO as number)) {
             supportedNetworks.reverse()
         }
         
         const [origin, destination] = supportedNetworks

        return { origin, destination }
}, [chainId])

Reply:
that doesnt seem right at all, supportednetworks is not supposed to be only fuse/celo

Reply:
I have not seen or heard a single thing about it supporting eth bridging.
Or is it an 'in case we want to enable' sort of comment? @sirpy 

Small side-comment that we have SupportedNetworks (all networks)
and we have SupportedV2Networks (used for the most recent or soon-to-be-released features)
Maybe we should rename it to SupportedAlts or something...

Reply:
anyway, looked for an alternative (as I have now seen the other networks which could potentially be enabled later)

But as of now, there is no way to determine the target network selected, but have put this request forward to them
kept a local bridge networks object instead until that is fixed